tenancy-api-mapping/openapispecs/generated/amc-infra-core-edge-infrastructure-manager-openapi-all.yaml

```diff
@@ -2,101 +2,84 @@
 # SPDX-FileCopyrightText: 2025 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-openapi: 3.0.2
+openapi: 3.0.3
 info:
   title: Edge Infrastructure Manager
-  description: Edge Infrastructure Manager API
-  license:
-    name: Apache 2.0
-    url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.1.0-dev
+  description: Edge Infrastructure Manager API - License Apache 2.0
```

Comment:
Should this license information be in a separate set of lines, like earlier?

Reply:
This was added due to the limitation of the plugin to auto generate the OpenAPI spec.